azuredevops.go

```diff
@@ -106,6 +112,20 @@ func (m importModule) runAzureDevOpsHandler(c *gin.Context) {
 	c.Status(http.StatusCreated)
 }
 
+func parseRepoRefParams(wharfGroupName, wharfProjectName string) (azureOrgName, azureProjectName, azureRepoName string) {
+	slashIndex := strings.IndexRune(wharfGroupName, '/')
+	if slashIndex == -1 {
+		azureOrgName = wharfGroupName
+		azureProjectName = wharfProjectName
+		azureRepoName = ""
+		return
+	}
+	azureOrgName = wharfGroupName[:slashIndex]
+	azureProjectName = wharfGroupName[slashIndex+1:]
```

Comment:
The 1 here is a magic number- refactor to named const for better readability.

Reply:
Ex `nameStartIndex`

Reply:
Reworked the function, as mentioned in https://github.com/iver-wharf/wharf-provider-azuredevops/pull/31#discussion_r701124201